lenovo/thinkpad/p16s/intel/default.nix

```diff
@@ -0,0 +1,8 @@
+{modulesPath, lib, ...}: {
+  imports = [
+    (modulesPath + "/installer/scan/not-detected.nix")
```

Comment:
nixos-hardware depends on nixos-generate-config. We try to not replace parts that can be auto-generated.

```suggestion
```